doc/api/crypto.md

```diff
@@ -1241,18 +1241,14 @@ input.on('readable', () => {
 added: v0.1.92
 -->
 
-Creates and returns a `Sign` object that uses the given `algorithm`. On
-recent OpenSSL releases, `openssl list-public-key-algorithms` will
-display the available signing algorithms. One example is `'RSA-SHA256'`.
+Creates and returns a `Sign` object that uses the given `algorithm`. For available signing algorithms, please reference `crypto.getHashes()`.
```

Comment:
If we go with this approach, it would be best to make `crypto.getHashes()` a link to the right location.